### PR TITLE
feat: implemented to_dense() to paddle frontend

### LINKED ISSUE #26823: to_dense

### PR DESCRIPTION
**Pull request by Mac16661**
Closes #26823 

- [x] Did you add a function?
- [x] Did you add the tests?
- [ ] Did you run your tests and are your tests passing?
- [x] Did pre-commit not fail on any check?
- [x] Did you follow the steps we provided?

### CODE REVIEW

File: ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_tensor.py [hunk 1]
```diff
@@ -151,29 +151,6 @@ def _get_dtype_and_square_matrix(draw):
     return dtype, mat
 
 
-# bmm helper function
-@st.composite
```
Comment (juliagsy) [inline ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_tensor.py:155]: hey! thanks for contributing! please don't remove anything from the file. you should only add the test for the function you're working on, thanks!

**Comment by Mac16661** [comment]
Actually I didn't  removed any thing. I have just added the to_dense test
function. May be I did git pull that's why some code from the file is
removed.

File: ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_tensor.py (hunk 1 above)
Comment (Mac16661) [reply ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_tensor.py:155]: I made a git pull request . I think that's why  some line of code got deleted

**Comment by juliagsy** [comment]
ahh but the PR diff is showing the removes, and they will be pushed altogether if we were to merge, could you please make sure that your fork+branch are up to date with the latest commit on master (then include your frontend function changes) and either update the PR again or open a new one if the latter simplifies things? Thanks!

**Comment by Mac16661** [comment]
feat(paddle_frontend): created a new pull request for to_dense method and test function #27040

**Pull request closed without merging**